oneBarangay/static/assets/js/volt.js

```diff
@@ -183,75 +183,75 @@ d.addEventListener("DOMContentLoaded", function (event) {
         plugins: [Chartist.plugins.tooltip()],
         axisX: {
           // On the x-axis start means top and end means bottom
-          position: "end",
-          showGrid: true,
+          position: 'end',
+          showGrid: true
         },
         axisY: {
           // On the y-axis start means left and end means right
           showGrid: false,
           showLabel: false,
           labelInterpolationFnc: function (value) {
-            return "$" + value / 1 + "k";
-          },
-        },
+            return '$' + value / 1 + 'k'
+          }
+        }
       }
-    );
+    )
   }
 
-  if (d.querySelector(".ct-chart-ranking")) {
+  if (d.querySelector('.ct-chart-ranking')) {
     const chart = new Chartist.Bar(
-      ".ct-chart-ranking",
+      '.ct-chart-ranking',
       {
-        labels: ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat"],
+        labels: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat'],
         series: [
           [1, 5, 2, 5, 4, 3],
-          [2, 3, 4, 8, 1, 2],
-        ],
+          [2, 3, 4, 8, 1, 2]
+        ]
       },
       {
         low: 0,
         showArea: true,
         plugins: [Chartist.plugins.tooltip()],
         axisX: {
           // On the x-axis start means top and end means bottom
-          position: "end",
+          position: 'end'
         },
         axisY: {
           // On the y-axis start means left and end means right
           showGrid: false,
           showLabel: false,
-          offset: 0,
-        },
+          offset: 0
+        }
       }
-    );
+    )
 
-    chart.on("draw", function (data) {
-      if (data.type === "line" || data.type === "area") {
+    chart.on('draw', function (data) {
+      if (data.type === 'line' || data.type === 'area') {
         data.element.animate({
           d: {
             begin: 2000 * data.index,
             dur: 2000,
             from: data.path.clone().scale(1, 0).translate(0, data.chartRect.height()).stringify(),
             to: data.path.clone().stringify(),
-            easing: Chartist.Svg.Easing.easeOutQuint,
-          },
-        });
+            easing: Chartist.Svg.Easing.easeOutQuint
+          }
+        })
       }
-    });
+    })
   }
 
-  if (d.querySelector(".ct-chart-traffic-share")) {
+  if (d.querySelector('.ct-chart-traffic-share')) {
     const data = {
-      series: [70, 20, 10],
-    };
+      series: [70, 20, 10]
+    }
 
     const sum = function (a, b) {
-      return a + b;
-    };
+      return a + b
+    }
 
-    new Chartist.Pie(".ct-chart-traffic-share", data, {
+    new Chartist.Pie('.ct-chart-traffic-share', data, {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PrynsTag/oneBarangay/pullRequest?prid=7999497)
```suggestion
          new Chartist.Pie(".ct-chart-traffic-share", data, {
```